test/Microsoft.ML.Tests/Transformers/HashTests.cs

```diff
@@ -124,7 +125,26 @@ public void TestOldSavingAndLoading()
             }
         }
 
-        private void HashTestCore<T>(T val, PrimitiveDataViewType type, uint expected, uint expectedOrdered, uint expectedOrdered3)
+        [Fact]
+        public void TestOldFormatModel()
+        {
+            string dataPath = GetDataPath("breast-cancer.txt");
```

Comment:
I think it would be better to increment the version again as I explained in the other comment, so I will leave this test.
(the existing test may be enough because the actual serialization format didn't change between versions 0x00010002 and 0x00010003, but I think it's better to have too much testing than too little testing :-). I did change he version of the old model from 0x00010002 to 0x00010003, so the tests are not identical). 

---
In reply to: [424674791](https://github.com/dotnet/machinelearning/pull/4828#discussion_r424674791) [](ancestors = 424674791)